src/test/clojure/clara/generative_tests/generators.clj

```diff
@@ -0,0 +1,102 @@
+(ns clara.generative-tests.generators
```

Comment:
It seems like the word "tests" is redundant since this is already under src/test. 

I'd just recommend
`clara.generative.generators`